### PR TITLE
Add license field to package metadata

### PR DESCRIPTION
**Pull request by earlbread**
This patch adds the license field to package metadata. (#100)

### CODE REVIEW

File: src/Nirum/Package/Metadata.hs [hunk 1]
```diff
@@ -96,11 +97,12 @@ deriving instance (Ord t, Target t) => Ord (Package t)
 deriving instance (Show t, Target t) => Show (Package t)
 
 packageTarget :: Target t => Package t -> t
-packageTarget Package { metadata = Metadata _ _ _ t } = t
+packageTarget Package { metadata = Metadata _ _ _ _ t } = t
```
Comment (dahlia) [inline src/Nirum/Package/Metadata.hs]: As `Metadata` get having many fields, we'd better use record syntax:

```haskell
packageTarget Package { metadata = Metadata { target = t } } = t
```

Comment (earlbread) [reply src/Nirum/Package/Metadata.hs]: I agree with you, I'll fix it soon. Thank you!

**Comment by dahlia** [comment]
Although builds were failed, it's not caused by this patch.  These failures have occasionally happened, so never mind.

**Comment by earlbread** [comment]
I updated the code using record syntax.

**Comment by dahlia** [comment]
@earlbread Thanks for your contribution. 😄 Just merged!

**Comment by earlbread** [comment]
@dahlia Thank you!

**Pull request merged**